ckanext/xloader/loader.py

```diff
@@ -398,9 +427,15 @@ def load_table(table_filepath, resource_id, mimetype='text/csv', logger=None):
                 'timestamp': datetime.datetime,
             }.get(existing_info.get(h, {}).get('type_override'), t)
             for t, h in zip(types, headers)]
+        for h in headers:
+            fields.append(existing_fields_by_headers.get(h, {}))
+    else:
+        # default strip_extra_white
```

Comment:
There appears to be no way to override this default before CKAN 2.11.

Reply:
@ThrawnCA okay I figured out how to do it pre `IDataDictionaryForm` implement. It just goes into `info__` in the template which gets saved into the info/comments for the column. I have been notified that this is not the best as there is no validation that can happen there (hence why the template now has a nasty `!= 'False'` as the info field would get saved as text). But that is how the current (2.10) CKAN works, so that is why the `IDataDictionaryForm` is important for the future.

That being said, I did put in the stuff in the template to allow for the override field of `strip_extra_white`, with some minor changes to the `loader.py` script now as I had to check for existing `strip_extra_white` in the info instead of fields for `<=2.10` versions. And this would allow the `strip_extra_white` to pertain for the columns when someone upgrades to `2.11`